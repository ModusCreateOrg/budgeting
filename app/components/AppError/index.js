import React from 'react';
import styles from './style.scss';

const AppError = () => (
  <div className={styles.errorScreen}>
    <div className={styles.errorContainer}>
      <h1 className={styles.errorTitle}>{`Sorry, something went wrong.`}</h1>
      <p>{`We're working on it and we'll get it fixed as soon as we can.`}</p>
    </div>
  </div>
);

export default AppError;
