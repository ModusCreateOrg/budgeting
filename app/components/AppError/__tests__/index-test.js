import React from 'react';
import renderer from 'react-test-renderer';
import AppError from '../';

it('renders correctly', () => {
  const tree = renderer.create(<AppError />).toJSON();
  expect(tree).toMatchSnapshot();
});
