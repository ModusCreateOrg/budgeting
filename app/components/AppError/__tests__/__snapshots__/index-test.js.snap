// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="errorScreen"
>
  <div
    className="errorContainer"
  >
    <h1
      className="errorTitle"
    >
      Sorry, something went wrong.
    </h1>
    <p>
      We're working on it and we'll get it fixed as soon as we can.
    </p>
  </div>
</div>
`;
